README.md

```diff
@@ -83,6 +83,9 @@ Swiping the trackpad horizontally with three fingers will scroll the tiling (onl
 
 PaperWM doesn't handle attached modal dialogs very well, so it's best to turn it off in Gnome Tweaks (under Windows).
 
+> **NOTE**
+> A summary of all current default Keybindings can be found in [Keybindings.md](Keybindings.md)
```

Comment:
Looks like a typo, note lack of camel-case (actual file name is camel-case).

This will break the link on rendering in github (i.e. go test that link in for fork page in github).

Reply:
fixed